iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift

```diff
@@ -202,7 +202,7 @@ final class PlacePagePreviewViewController: UIViewController {
         details = nil;
       }
       
-      setScheduleLabel(state: L("closed_now"),
+      setScheduleLabel(state: L(""),
```

Comment:
Is it from another PR?

Reply:
yes

Reply:
this  is the PR belong to this isssue https://github.com/organicmaps/organicmaps/pull/4491/commits/cbfc515a8c821f5042f8341dc5eb2b10ef095db5

Reply:
this PR is cbfc515a8c821f5042f8341dc5eb2b10ef095db5